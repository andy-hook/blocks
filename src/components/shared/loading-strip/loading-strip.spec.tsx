import React from "react"

import LoadingStrip from "./loading-strip"
import { renderWithTheme } from "@test-utils"

describe("<LoadingStrip />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <LoadingStrip loading={true} />)

    expect(tree).toMatchSnapshot()
  })
})
