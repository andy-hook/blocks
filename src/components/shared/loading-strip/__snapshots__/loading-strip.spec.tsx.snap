// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingStrip /> renders correctly 1`] = `
<div
  className="sc-cSHVUG abUMB"
/>
`;
